### PR TITLE
Fix Wyoming scraper

### LINKED ISSUE #174: Wyoming has a KeyError
An error occurred. ... KeyError('ATTR(Date Updated)')

### PR DESCRIPTION
**Pull request by fredtsun**
Fixes #174 

The schema for the Wyoming Tableau data have changed and this PR has some adjusted strings to account for them.

### CODE REVIEW

File: workflow/python/covid19_scrapers/states/wyoming.py
Comment (nkrishnaswami) [inline workflow/python/covid19_scrapers/states/wyoming.py:33]: Is this fragile? Would it be better to wait for content instead of count?

Comment (nkrishnaswami) [reply workflow/python/covid19_scrapers/states/wyoming.py:33]: Or per the other PR, do we still need a `wait_` here?

**Review by nkrishnaswami — CHANGES_REQUESTED** [review]

File: workflow/python/covid19_scrapers/states/wyoming.py
Comment (fredtsun) [reply workflow/python/covid19_scrapers/states/wyoming.py:33]: Yeah thats a good point, definitely fragile. But this is waiting for elements!

And for the second part, the behavior here is a little weird in general. I'm actually gonna hold off on this one and the other one.

What I'm observing is that when a Tableau request returns, sometimes the body doesn't have a complete set of data. So the driver code thinks that the request is complete, but there are still things being returned(?!). I'm not exactly sure why that is but I'm going to dig a bit more (this whole thing I'm observing doesn't make much sense to me from a request/response pov).

But what this means is that for Tableau requests, waiting for the elements to be rendered ensures that the complete body is returned. From what I've seen, PowerBI doesn't experience this kind of behavior (the body is always complete).

I'm thinking that I want to bake some methods into the webdriver steps. Something like `find_tableau_request` and to be complete I think I'll also add a `find_powerbi_request`. That way the library code can do the proper validations and the webdriver doesn't need to worry about that. I think the interface for the webdriver runner will be a bit cleaner as a result. The con here is that I adding Tableau

Comment (nkrishnaswami) [reply workflow/python/covid19_scrapers/states/wyoming.py:33]: Sounds good, thanks -- do you prefer to merge these and do that in a new PR, or make the change you describe first?

Comment (fredtsun) [reply workflow/python/covid19_scrapers/states/wyoming.py:33]: Hmm, I think it might be more helpful to merge first to unblock the scrapers? What do you think?

Comment (nkrishnaswami) [reply workflow/python/covid19_scrapers/states/wyoming.py:33]: 👍

**Pull request merged**
